code/__DEFINES/antagonists.dm

```diff
@@ -81,3 +81,4 @@
 #define PATH_ASH "Ash"
 #define PATH_RUST "Rust"
 #define PATH_FLESH "Flesh"
+#define COMSIG_VOID_MASK_ACT "void_mask_act"
```

Comment:
I'd advise renaming this signal to `COMSIG_HUMAN_VOID_MASK_ACT`. (`HUMAN` means it only can be signaled to humans)

And you *should* move it to code/_DEFINES/components.dm.